plugin/db/mongodb/mongodb.go

```diff
@@ -65,14 +72,56 @@ func (driver *Driver) Ping(ctx context.Context) error {
 	return nil
 }
 
+// GetType returns the database type.
+func (*Driver) GetType() db.Type {
+	return db.MongoDB
+}
+
 // GetDBConnection returns a database connection.
 func (*Driver) GetDBConnection(_ context.Context, _ string) (*sql.DB, error) {
-	panic("not implemented")
+	return nil, nil
```

Comment:
return nil, err ?

Reply:
Or comment otherwise.

Reply:
em, then this sounds like it should never be called, and panic("not implemented") actually makes sense. Because if it gets called, it's a logic error?

Reply:
Done, return an error, and then if any new code call this function via driver interface, we can detect this error quickly.